variables.tf

```diff
@@ -79,3 +79,62 @@ variable "server_name" {
   description = "The name of the server"
   default     = "default"
 }
+
+variable "cluster_ingress_hostname" {
+  type        = string
+  description = "Ingress hostname of the IKS cluster."
+  default     = ""
+}
+
+variable "operator_namespace" {
+  type        = string
+  description = "operator namespace"
+  default     = "cpd-operators"
+}
+
+variable "cpd_namespace" {
+  type        = string
+  description = "cpd namespace"
+  default     = "gitops-cp4d-instance"
+}
+
+variable "service_account_name" {
+  type        = string
+  description = "The name of the service account for wkc"
+  default     = "wkc-iis-sa"
+}
+
+variable "sccs" {
+  type        = list(string)
+  description = "The list of sccs that should be generated for the service account (valid values are anyuid and privileged)"
+  default     = ["anyuid"]
+}
+
+variable "rbac_label" {
+  type        = string
+  description = "The name for RBAC rule"
+  default     = "wkc-iis-scc-rb"
+}
+
+variable "rbac_rules" {
```

Comment:
We don't need to expose this variable either

Reply:
implemented the change